guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,400 @@
+
+Silent payment wallets may or may not be set up as [BIP-39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) and/or [BIP-32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) HD wallets since the protocol does not require these.
+
+
+## Sending
+
+With the improvements to Contacts, on-chain send flows may start from a number of [entry points](/guide/daily-spending-wallet/sending/#payment-entry-points). When users start with obtaining a static address, every on-chain address derived from it will be visibly different from the static address the sender started with. This is likely to be confusing for users. Applications should take measures such as short explainers to avoid confusion on the users part. Test transactions are another good way to help users deal with this.
+
+{% include image-gallery.html pages = page.images_send %}
+
+
+{% include tip/recommendation.html %}
+
+Applications that do not support static addresses should provide helpful, actionable, human readable errors.
+
+{% include tip/close.html %}
+
+### Coin selection
+As mentioned in the introduction, coin selection can be done much better due to auto-applied label information. Applications should encourage & assist users in performing coin selection by surfacing relevant or related labels or other methods. Automatic coin selection should be improved with all the label information available.
+
+{% include tip/tip.html %}
+
+The coin(s) selected for a transaction determine the derived on-chain address due to the nature of silent payments protocol. The interface should avoid showing the on-chain address before the coin selection is done.
+
+{% include tip/close.html %}
+
+## Receiving & scanning
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/static-address-with-customization-options.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/static-address-with-customization-options@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/static-address-with-customization-options.png"
+   width = 250
+   height = 522
+   alt-text = "The user contact page listing their receive static address along with the ability to add labels before sharing it."
+   caption = "The user contact page allows the user to add labels to their receive static address before sharing it."
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Receiving flows are likely to be used less often since static addresses can be safely reused. If the receiver has publicly posted their static address, they will receive payments without any interaction with the sender. Therefore, applications should encourage users to add labels or a contact whenever they actively share a static address with a sender.
+
+Applications should also allow receivers to share an on-chain address itself in case the sender's wallet cannot send to static addresses.
```

Comment:
```suggestion
Applications should also allow receivers to generate and share conventional on-chain addresses in case the sender's wallet cannot send to static addresses.
```